src/components/nav/Nav.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation: React.FC = () => (
```

Comment:
The base filename 'Nav.tsx' does not match the name of its primary named or default export 'Navigation'. Consider renaming the file to 'Navigation.tsx' or updating the export to 'Nav'.